src/config/config.cc

```diff
@@ -159,6 +159,7 @@ Config::Config() {
       {"log-retention-days", false, new IntField(&log_retention_days, -1, -1, INT_MAX)},
       {"persist-cluster-nodes-enabled", false, new YesNoField(&persist_cluster_nodes_enabled, true)},
       {"redis-cursor-compatible", false, new YesNoField(&redis_cursor_compatible, false)},
+      {"repl-namespace-enabled", false, new YesNoField(&repl_namespace_enabled, false)},
```

Comment:
What will happen if the master enables it but the replica does not, or vice versa?

Reply:
This option is only to control if the current instance needs to replicate its namespace, so if the master enables it but slaves do not, then namespaces will be still replicated to slaves. And vice versa, all namespaces won't be replicated in master and slaves, since we don't allow changing namespace if the option is enabled and its role is slave.

Reply:
got it.